### PR TITLE
Update PennyLane-SF for breaking changes in SF v0.11

### PR DESCRIPTION
**Pull request by josh146**
**Description of the Change:** Updates the PennyLane-SF plugin for the breaking changes introduced in SF version 0.11

**Benefits:** PennyLane-SF now works with latest development version of SF.

**Possible Drawbacks:** n/a

**Related GitHub Issues:** n/a

### CODE REVIEW

**Comment by josh146** [comment]
This has uncovered a bug in SF on Python 3.5, as `ModuleNotFoundError` is not defined. However, the tests pass on 3.6.

**Comment by josh146** [comment]
@co9olguy, this is now ready for review. I've temporarily moved travis from testing against python 3.5 and 3.6 to 3.6 and 3.7 to avoid this bug, as it will instead need to be fixed in SF.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-strawberryfields>=0.9
-pennylane>=0.4
+strawberryfields
```
Comment (co9olguy) [inline requirements.txt]: no versions?

File: setup.py [hunk 2]
```diff
@@ -22,8 +22,8 @@
 
 
 requirements = [
-    "strawberryfields>=0.9",
-    "pennylane>=0.4"
+    "strawberryfields",
```
Comment (co9olguy) [inline setup.py]: same question

File: requirements.txt (hunk 1 above)
Comment (josh146) [reply requirements.txt]: You're right, this is an oversight on my end.
```suggestion
strawberryfields>=0.11
```

Comment (josh146) [reply requirements.txt]: Ah, I remember now. Travis is set to install SF from github master - setting >=0.11 before 0.11 was released was causing pip to fail, so I temporarily removed version pinning.

File: setup.py (hunk 2 above)
Comment (josh146) [reply setup.py]:
```suggestion
    "strawberryfields>=0.11",
```

File: requirements.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-strawberryfields>=0.9
-pennylane>=0.4
+strawberryfields>=0.11
+pennylane
```
Comment (josh146) [inline requirements.txt]:
```suggestion
pennylane>=0.4
```

File: setup.py [hunk 4]
```diff
@@ -22,8 +22,8 @@
 
 
 requirements = [
-    "strawberryfields>=0.9",
-    "pennylane>=0.4"
+    "strawberryfields>=0.11",
+    "pennylane"
```
Comment (josh146) [inline setup.py]:
```suggestion
    "pennylane>=0.4"
```

**Pull request merged**